src/admin-views/aggregator/settings.php

```diff
@@ -112,7 +112,7 @@
 
 $global = $ical = $ics = $facebook = $gcal = $meetup = $url = array();
 // if there's an Event Aggregator license key, add the Global settings, Facebook, iCal, and Meetup fields
-if ( Tribe__Events__Aggregator::is_service_active() ) {
+if ( Tribe__Events__Aggregator::is_service_active() && ! empty( get_option( 'pue_install_key_event_aggregator' ) ) ) {
```

Comment:
We need to setup a method to get the _PUE License_ due to been a a little bit more complex than this.
See this code for why is that:
https://github.com/moderntribe/the-events-calendar/blob/master/src/Tribe/Aggregator/Service.php#L82-L86

Side note, avoid `empty()` directly to a method or function, it can only be called to variables.